datumaro/components/extractor.py

```diff
@@ -517,9 +581,40 @@ class DatasetItem:
         type=str, validator=not_empty)
     annotations = attrib(factory=list, validator=default_if_none(list))
     subset = attrib(converter=lambda v: v or DEFAULT_SUBSET_NAME, default=None)
+
+    # Currently unused
     path = attrib(factory=list, validator=default_if_none(list))
 
+    # TODO: introduce "media" field with type info. Replace image and pcd.
     image = attrib(type=Image, default=None)
+    pcd = attrib(type=bytes, default=None)
```

Comment:
Perhaps for clarity it would be better to use a longer name, e.g. `point_cloud`?

Reply:
Done